.dockerignore

```diff
@@ -1,4 +1,4 @@
-.git
+#.git
```

Comment:
I am a minor concerned about loading `.git` into docker containers. This might become quite slow 🤔